test/rootful.bats

```diff
@@ -38,3 +38,25 @@ skipIfDockerUnavailableAndNotRunAsRoot() {
 	skipIfDockerUnavailableAndNotRunAsRoot
 	testPortForwarding -u root:root -v "$PODMAN_ROOT_DATA_DIR:/var/lib/containers/storage" "${PODMAN_IMAGE}"
 }
+
+@test "$TEST_PREFIX quedlet - generate service" {
```

Comment:
There seems to be a typo here: 'quedlet' should be changed to 'quadlet'.

Same for the other new test case you added within `rootless.bats`.